auth/README.md

```diff
@@ -67,7 +68,7 @@ locally:
 version: "3.7"
 services:
   auth:
-    image: mainflux/auth:[version]
+    image: mainflux/auth:[MF_RELEASE_TAG]
```

Comment:
I don't get this, why it's wrapped with those brackets `[MF_RELEASE_TAG]`? I was expecting the same syntax as in Yaml `${MF_RELEASE_TAG}` or at least `<MF_RELEASE_TAG>` a syntax used for variables in doc.

Reply:
or maybe `[Docker image version tag specified by MF_RELEASE_TAG]`

Reply:
I would go with `${MF_RELEASE_TAG}` because its env var and exact syntax. But yea, it's probably better because those brackets are everywhere.